client/src/boot/index.js

```diff
@@ -8,6 +8,7 @@ import { setConfig } from 'state/config/ConfigActions';
 import registerComponents from 'boot/registerComponents';
 import registerReducers from 'boot/registerReducers';
 import applyDevtools from 'boot/applyDevtools';
+import transforms from './transforms';
```

Comment:
import from `boot/transforms` would be following existing conventions, could this be named `applyTransforms`